x/superfluid/keeper/internal/events/emit.go

```diff
@@ -62,6 +62,24 @@ func newSuperfluidDelegateEvent(lockId uint64, valAddress string) sdk.Event {
 	)
 }
 
+func EmitCreateFullRangePositionAndSuperfluidDelegateEvent(ctx sdk.Context, lockId uint64, valAddress string) {
+	if ctx.EventManager() == nil {
+		return
+	}
+
+	ctx.EventManager().EmitEvents(sdk.Events{
+		newCreateFullRangePositionAndSuperfluidDelegateEvent(lockId, valAddress),
+	})
+}
+
+func newCreateFullRangePositionAndSuperfluidDelegateEvent(lockId uint64, valAddress string) sdk.Event {
+	return sdk.NewEvent(
+		types.TypeEvtCreateFullRangePositionAndSFDelegate,
+		sdk.NewAttribute(types.AttributeLockId, osmoutils.Uint64ToString(lockId)),
+		sdk.NewAttribute(types.AttributeValidator, valAddress),
```

Comment:
We might want to return the position ID here as well

Reply:
Good catch, added